index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Product Landing Page</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <div>
+      <header id="header">
+        <div>
+          <img
+            id="header-img"
+            src="https://pngimg.com/uploads/trumpet_saxophone/small/trumpet_saxophone_PNG14767.png"
+            alt="Trumpet image"
+            width="160"
+            height="70"
+          />
+        </div>
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#something">Features</a></li>
+            <li><a class="nav-link" href="#somethingmore">How It Works</a></li>
+            <li><a class="nav-link" href="#evenmore">Prices</a></li>
+          </ul>
+        </nav>
+      </header>
+      <div>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            name="email"
+            id="email"
+            type="email"
+            placeholder="Enter your email address"
+            required=""
+          />
+          <input name="submit" id="submit" type="submit" value="Submit" />
+        </form>
+      </div>
+
+      <section id="something">
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/1350/1350727.png"
+              alt="Materials"
+              width="60"
+              height="70"
+            />
+          </div>
+          <div class="descrip">
+            <h2>Premium Materials</h2>
+            <p>
+              Our trumpets use the shiniest brass which is sourced locally. This
+              will increase the longevity of your purchase.
+            </p>
+          </div>
+        </div>
+        <div>
+          <div class="icon">
+            <img
+              src="https://cdn-icons-png.flaticon.com/512/5770/5770775.png"
+              alt="Materials"
+              width="70"
+              height="70"
+            />
+          </div>
+
+          <div class="descrip">
+            <h2>Fast Shipping</h2>
+            <p>
```

Comment:
Change the id names to something more descriptive

Reply:
**_Change the id names to something more descriptive_**

  

- [ ] I think it would be better to make it a bit clear with the specific `id` being referred to. The `id` word could be any word in the code so I suggest being specific by saying the `id` selector inside the `section` tag.
- [ ] I suggest the comment should be more politely written by saying **_"I think it would be better ...."_**
- [ ] I think line `42` could only be highlighted rather than highlighting from line `42` to `72` to perfectly be able to identify the line of code the comment is made.